tests/sentry/sentry_metrics/test_multiprocess_steps.py

```diff
@@ -260,8 +260,7 @@ def __translated_payload(
     return payload
 
 
-@patch("sentry.sentry_metrics.consumers.indexer.processing.get_indexer", return_value=MockIndexer())
```

Comment:
since https://github.com/getsentry/sentry/pull/36397 this is no longer necessary. the global indexer is mocked and cleared on every testrun because of the `sentry_metrics` marker, because this exact decorator was scattered across hundreds of product/api tests as well it seemed like a good default